### ISSUE TITLE
v5 beta - missing test times in summary in console

### ISSUE THREAD

**Issue by lukeapage** [cypress-io/cypress#8309]
I've been trying out the latest master builds to see if I can get our tests upgraded to v5 and noticed this - found the same problem on your own builds.

Latest master branch build on windows:

https://ci.appveyor.com/project/cypress-io/cypress/builds/34714208/job/ip9ijo360b0w3ke5?fullLog=true

line 2080:
```
       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  examples\actions.spec.js                   0ms       14       14        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\aliasing.spec.js                  0ms        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\assertions.spec.js                0ms        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\connectors.spec.js                0ms        8        8        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\cookies.spec.js                   0ms        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\cypress_api.spec.js               0ms       13       13        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\files.spec.js                     0ms        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\local_storage.spec.js             0ms        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\location.spec.js                  0ms        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\misc.spec.js                      0ms        6        6        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\navigation.spec.js                0ms        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\network_requests.spec.js          0ms        7        7        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\querying.spec.js                  0ms        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\spies_stubs_clocks.spec.js        0ms        7        7        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\traversal.spec.js                 0ms       18       18        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\utilities.spec.js                 0ms        6        6        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\viewport.spec.js                  0ms        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\waiting.spec.js                   0ms        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  examples\window.spec.js                    0ms        3        3        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        02:26      117      117        -        -        -  
yarn test-mocha
```

Notice how the times for each individual spec are 0ms - this used to show correct, not sure when it broke, we're stuck on 4.4.1 for performance reasons.

**Comment by bahmutov** [comment]
Good eye @lukeapage thanks for reporting this

Individual spec has test durations

```

  (Run Starting)
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Cypress:    4.12.1                                                                             │
  │ Browser:    Chrome 84                                                                          │
  │ Specs:      19 found (examples\actions.spec.js, examples\aliasing.spec.js, examples\assertions │
  │             .spec.js, examples\connectors.spec.js, examples\cookies.spec.js, examples\cypress_ │
  │             api.spec.js, examples\files.spec.js, examples\local_storage.spec.js, examples\loca │
  │             tion.spec.j...)                                                                    │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
────────────────────────────────────────────────────────────────────────────────────────────────────
                                                                                                    
  Running:  examples\actions.spec.js                                                       (1 of 19)
  Actions
    √ .type() - type into a DOM element (7573ms)
    √ .focus() - focus on a DOM element (628ms)
    √ .blur() - blur off a DOM element (964ms)
    √ .clear() - clears an input or textarea element (974ms)
    √ .submit() - submit a form (956ms)
    √ .click() - click on a DOM element (3009ms)
    √ .dblclick() - double click on a DOM element (646ms)
    √ .rightclick() - right click on a DOM element (763ms)
    √ .check() - check a checkbox or radio element (1608ms)
    √ .uncheck() - uncheck a checkbox element (1428ms)
    √ .select() - select an option in a <select> element (1503ms)
    √ .scrollIntoView() - scroll an element into view (609ms)
    √ .trigger() - trigger an event on a DOM element (482ms)
    √ cy.scrollTo() - scroll the window or element to a position (2595ms)
  14 passing (28s)
  (Results)
```

But the specs summary table at the end does not

**Comment by bahmutov** [comment]
Same missing timestamps on Linux https://app.circleci.com/pipelines/github/cypress-io/cypress-test-example-repos/4619/workflows/855356a6-8da9-4697-9001-823afcd8d2cd/jobs/118576

**Comment by bahmutov** [comment]
before v5 merge https://github.com/cypress-io/cypress/commit/14e13f39a42da171f3a0431a9a14b71016e212dc is working

Broken in v5 merge commit https://github.com/cypress-io/cypress/pull/7753

working in https://github.com/cypress-io/cypress/pull/7753/commits/27e8c817d34a9ac70ff840d45938d16d3a225736

working in https://github.com/cypress-io/cypress/pull/7753/commits/09c72e3589cc64316b8b6aa3f34462a661a19474

**Comment by bahmutov** [comment]
working in https://github.com/cypress-io/cypress/pull/7753/commits/87029a7f3e426cf2d104aa71849d6eeb74412e3e

working in https://github.com/cypress-io/cypress/pull/7753/commits/f65a50f411de12cc30e30027fda4ae1d61c85478

**Comment by bahmutov** [comment]
working in https://github.com/cypress-io/cypress/pull/7753/commits/01cc5e96d005c09c6972563c05e285dd0e825e99 

broken in commit that removes private properties from test results https://github.com/cypress-io/cypress/pull/7753/commits/966fbf0e421ba99bb0df5845493fd1842654323a

**Comment by jennifer-shehane** [comment]
Released in [`5.0.0`](https://on.cypress.io/changelog#5-0-0).

This comment thread has been locked. If you are still experiencing this issue after upgrading to Cypress v5.0.0, please open a new issue.
